rubocop/rubocop.yml

```diff
@@ -169,3 +169,186 @@ Layout/EmptyLineAfterGuardClause:
 
 Migration/DepartmentName:
   Enabled: false
+
+# The following cops were added between 0.92.0 and 1.14.0
```

Comment:
These are all new cops that were added or enabled since Rubocop 0.92.0. I've gone ahead and disabled them until further review.